WooCommerce/WooCommerceTests/ViewRelated/Shipping Label/Reprint Shipping Label/PrintShippingLabelCoordinatorTests.swift

```diff
@@ -94,36 +94,6 @@ final class PrintShippingLabelCoordinatorTests: XCTestCase {
         }
     }
 
-    func test_print_on_failure_presents_error_alert() throws {
```

Comment:
I cannot update the test to make it pass, somehow the completion closure is not triggered here. I guess it's ok to remove it.